.github/autobuild/mac.sh

```diff
@@ -68,6 +85,29 @@ pass_artifact_to_job() {
     echo "Moving build artifact to deploy/${artifact}"
     mv ./deploy/Jamulus-*installer-mac.dmg "./deploy/${artifact}"
     echo "::set-output name=artifact_1::${artifact}"
+
+    artifact2="jamulus_${JAMULUS_BUILD_VERSION}_mac${ARTIFACT_SUFFIX:-}.pkg"
+    if [ -f ./deploy/Jamulus_*.pkg ]; then
+        echo "Moving build artifact2 to deploy/${artifact2}"
+        mv ./deploy/Jamulus_*.pkg "./deploy/${artifact2}"
+        echo "::set-output name=artifact_2::${artifact2}"
+    fi
+}
+
+valid8_n_upload() {
+    echo ">>> Processing validation and upload..."
+    # test the signature of package
+    pkgutil --check-signature "${ARTIFACT_PATH}"
+    # attempt validate and then upload of pkg file, using previously-made keychain item
+    xcrun altool --validate-app -f "${ARTIFACT_PATH}" -t macos -u $NOTARIZATION_USERNAME -p $NOTARIZATION_PASSWORD
```

Comment:
Not sure about the quoting of the Password here.

Reply:
Values are output as `***`, nothing sensitive is written to log